xbmc/video/guilib/VideoSelectActionProcessor.cpp

```diff
@@ -70,7 +71,17 @@ bool CVideoSelectActionProcessorBase::Process(Action action)
       return OnQueueSelected();
 
     case ACTION_INFO:
+    {
+      if (GetDefaultAction() == ACTION_INFO && !KODI::VIDEO::IsVideoDb(*m_item) &&
+          !m_item->IsPlugin() && !m_item->IsScript() &&
+          !KODI::VIDEO::UTILS::HasItemVideoDbInformation(*m_item))
+      {
+        // for items without info fall back to resume/play
+        return OnResumeSelected();
```

Comment:
How about this instead, honoring the configured default play action?
```suggestion
        return Process( CVideoPlayActionProcessorBase::GetDefaultAction() );
```

Reply:
Yes, that's better. Will change.